doc_src/fish_indent.txt

```diff
@@ -7,12 +7,16 @@ fish_indent [OPTIONS]
 
 \subsection fish_indent-description Description
 
-`fish_indent` is used to indent a piece of fish code. `fish_indent` reads commands from standard input and outputs them to standard output.
+`fish_indent` is used to indent a piece of fish code. `fish_indent` reads commands from standard input and outputs them to standard output or a specified file.
 
 The following options are available:
 
 - `-d` or `--dump` dumps information about the parsed fish commands to stderr
 
+- `-w` or `--write` indents a specified file and immediatly writes to that file
```

Comment:
Drop the `-W` flag. It's redundant. The `-w` option handles the common use case in the manner of gofmt. Anyone wanting to send the output to a different file should just `fish_indent script.fish > new_script.fish`.